x-pack/plugins/gis/public/elasticsearch_geo_utils.js

```diff
@@ -231,47 +215,41 @@ export function convertMapExtentToEnvelope({ maxLat, maxLon, minLat, minLon }) {
   }
 
   if (maxLon > 180) {
-    // bounds cross datleine east to west, slit into 2 shapes
+    // bounds cross dateline east to west
     const overlapWestOfDateLine = maxLon - 180;
-    return [
-      convertMapExtentToEnvelope({
-        maxLat,
-        maxLon: 180,
-        minLat,
-        minLon,
-      }),
-      convertMapExtentToEnvelope({
-        maxLat,
-        maxLon: -180 + overlapWestOfDateLine,
-        minLat,
-        minLon: -180,
-      }),
-    ];
+    return convertMapExtentToPolygon({
+      maxLat,
+      maxLon: -180 + overlapWestOfDateLine,
+      minLat,
+      minLon,
+    });
   }
 
   if (minLon < -180) {
-    // bounds cross datleine west to east, slit into 2 shapes
+    // bounds cross dateline west to east, slit into 2 shapes
```

Comment:
remove comment, splitting is not really what we're doing (?). Rather, we're doing a union.